Excel-custom-functions/AzureFunction/README.md

```diff
@@ -0,0 +1,234 @@
+# Integrate an Azure function with your custom function
+
+You can expand the capabilities of Excel custom functions by integration with Azure functions. An Azure function allows you to move your code to the cloud so it is not visible from the browser, and you can choose additional languages to run in besides JavaScript. Also an Azure function can integrate with other Azure services such as message queues and storage. And you can share the function with other clients.
+
+This sample shows how to move the code from the `Add` custom function into an Azure function.
+
+## Applies to
+
+-  Excel, PowerPoint, Word on Windows, Mac, and in a browser.
+
+## Prerequisites
+
+To set up your development environment:
+
+1. Install [Visual Studio Code](https://code.visualstudio.com) on one of the supported platforms.
+2. Install version 2.x of the [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local#v2).
+3. Install the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp). You can also search for the C# extension in the VS Code Extensions view (Ctrl + Shift + X) and typing 'C# for Visual Studio Code'.
+4. Install the [Azure Functions extension](https://docs.microsoft.com/azure/azure-functions/functions-create-first-function-vs-code#install-the-azure-functions-extension). You can also search for the Azure Functions extension in the VS Code Extensions view (Ctrl + Shift + X) and typing 'Azure Functions'.
+
+You will also need the following:
+
+- An [Azure subscription](https://docs.microsoft.com/en-us/azure/guides/developer/azure-developer-guide#understanding-accounts-subscriptions-and-billing). You can create a [free account](https://azure.microsoft.com/free) before you begin.
+- An Office 365 account. You can join the [Office 365 Developer Program](https://aka.ms/devprogramsignup) that includes a free 1 year subscription to Office 365.
+
+## Solution
+
+Solution | Author(s)
+---------|----------
+Azure function integration | David Chesnut (**Microsoft**)
+
+## Version history
+
+Version  | Date | Comments
+---------| -----| --------
+1.0  | September 20, 2019 | Initial release
+
+## Disclaimer
+
+**THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.**
+
+----------
+## Try out the Azure function from localhost
+
+This sample moves the `Add` custom function to an Azure function named `AddTwo()`. The Azure function adds the two numbers and returns the result. The next steps will run the sample. If you want to know how it works, jump down to the explanation section.
+
+### Build and run the Azure function
+
+1. Clone or download this repository to a local project folder.
+2. Start Visual Studio Code and open the **AzureFunction/AzureFunctionProject** folder in the project.
+3. Open the tasks.json file and locate the entry for "type": "func".
```

Comment:
Please clarify the `.vscode` subfolder for this json file.

Reply:
Added full path.